test_markdown_writer.py

```diff
@@ -181,6 +182,7 @@ def test_write_to_markdown_with_vertical_bar_in_title(self):
             num_issues_opened=num_issues_opened,
             num_issues_closed=num_issues_closed,
             labels=["bug"],
+            report_title="Issue Metrics",
```

Comment:
We can leave it here so then it is tested as specified in one test and left to default in the other.